src/libs/runtime-utils.js

```diff
@@ -42,7 +42,10 @@ export const defaultLocation = 'US'
 export const defaultComputeZone = 'US-CENTRAL1-A'
 export const defaultComputeRegion = 'US-CENTRAL1'
 
-export const defaultAutoPauseThreshold = 30
+export const defaultAutopauseThreshold = 30
+export const defaultAutopauseDisabledValue = 0
```

Comment:
It's always the same value so we could drop `default` for concision:
```suggestion
export const autopauseDisabledValue = 0
```